src/core/src/shade.rs

```diff
@@ -21,8 +21,13 @@ use std::error::Error;
 use {Resources};
 use {AttributeSlot, ColorSlot, ConstantBufferSlot, ResourceViewSlot, SamplerSlot, UnorderedViewSlot};
 
-#[cfg(feature = "cgmath-types")]
-use cgmath::{Deg, Matrix2, Matrix3, Matrix4, Point2, Point3, Rad, Vector2, Vector3, Vector4};
+#[cfg(feature = "mint")]
+use mint::{
+    Point2, Point3,
```

Comment:
I don't think it makes sense to use all the types explicitly as opposed to a simple `use mint;`, given that all the usage is in macros, so there are no ergonomics concerns